public/_redirects

```diff
@@ -0,0 +1 @@
+/sui https://sui.portalbridge.com/ 301
```

Comment:
sure, I was not aware of it, let me push a new changeset

Reply:
I think the best is to keep siu.portalbridge.com because this is doing some proxy forward internally, if I change it by the redirect, the domain at the browser level will change, and we do not want that, I think, also because wormhole-connect-bridge.pages.dev does not look a production environment.